cmd/update.go

```diff
@@ -111,8 +111,11 @@ func updateGitops(gitopsName string, o *updateOptions) error {
 
 	// 3. Restart gitops and editor services
 	fmt.Println("Restarting services...")
-	dockerComposePath := filepath.Join(gitopsConfig, "deployment", "docker-compose")
-	err = exec.Command(dockerComposePath, "up").Run()
+	dockerComposePath := filepath.Join(gitopsConfig, "deployment")
+	cmd := exec.Command("sh", "-c", fmt.Sprintf("cd %s && docker-compose down && docker-compose up -d --remove-orphans", dockerComposePath))
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err = cmd.Run()
```

Comment:
Use dir and avoid using sh strings as they lead to bugs and security flaws https://pkg.go.dev/os/exec#Cmd